internal/types/api_errors.go

```diff
@@ -1,13 +1,18 @@
-package api
+package types
 
 // Copyright (C) 2024 by Posit Software, PBC.
 
 import (
+	"encoding/json"
 	"net/http"
-
-	"github.com/posit-dev/publisher/internal/types"
 )
 
+func jsonResult(w http.ResponseWriter, status int, result any) {
```

Comment:
Did we have to remake this from `api_helpers` due to the circular dependency?

Reply:
yes, that is correct. I didn't see a good way of moving the original JsonResult to somewhere else. Any suggestions?